### PR TITLE
Add new chain 7171 & 7771

### PR DESCRIPTION
**Pull request by BitrockChain**
# Add New Chain <chainId>

Thanks for your pull request to add a new support in Sourcify.

If you haven't done so, please follow the instructions on [how to request chain support](https://docs.sourcify.dev/docs/chain-support/) in docs.

Please check the following items before submitting your pull request for a speedy review.

## Checklist

- [x] The branch is named as `add-chain-<chainId>`.
- [x] I haven't modified the `chains.json` file directly.
- [x] In `sourcify-chains.json` file
  - [x] I've set `supported: true`.
  - [ ] I haven't added an `rpc` field but the one in [chains.json](../../src/chains.json) is used (if not, please explain why).
- [x] I've added a test in [chain-tests.js](../../test/chains/chains-test.js) file.
- [x] `test-new-chain` test in Circle CI is passing.

### CODE REVIEW

**Comment by manuelwedler** [comment]
Thanks for the PR. We still need to fix our tests ([#1734](https://github.com/ethereum/sourcify/issues/1734)), before we can merge this.

File: services/server/src/sourcify-chains-default.json [hunk 1]
```diff
@@ -2221,5 +2221,15 @@
...
+  },
+  "7171": {
+    "sourcifyName": "Bitrock Mainnet",
+    "supported": true,
+    "rpc": ["https://brockrpc.io/"]
+  },
+  "7771": {
+    "sourcifyName": "Bitrock Testnet",
+    "supported": true,
+    "rpc": ["https://testnet.bit-rock.io/"]
```
Comment (manuelwedler) [inline services/server/src/sourcify-chains-default.json:2233]: Please remove this

File: services/server/src/sourcify-chains-default.json [hunk 2]
```diff
@@ -2221,5 +2221,15 @@
     "sourcifyName": "Zircuit Testnet",
     "supported": true,
     "rpc": ["https://zircuit1-testnet.p2pify.com/"]
+  },
+  "7171": {
+    "sourcifyName": "Bitrock Mainnet",
+    "supported": true,
+    "rpc": ["https://brockrpc.io/"]
```
Comment (manuelwedler) [inline services/server/src/sourcify-chains-default.json:2228]: Please remove this

**Review by manuelwedler — CHANGES_REQUESTED** [review]
What I can tell already is that you can remove the rpc fields, because the RPCs are already inside chains.json

**Comment by BitrockChain** [comment]
Removed, thanks @manuelwedler

**Comment by BitrockChain** [comment]
> Thanks for the PR. We still need to fix our tests ([#1734](https://github.com/ethereum/sourcify/issues/1734)), before we can merge this.

Thanks, You’re the best!

**Comment by manuelwedler** [comment]
Your last commit removed the configuration changes completely. I added it again

**Review by manuelwedler — APPROVED** [review]

**Comment by BitrockChain** [comment]
hi @manuelwedler thanks for merged.

**Comment by manuelwedler** [comment]
@BitrockChain I'm disabling this chain as it has been constantly failing in our CI. Please reach out in case you still want to be supported by Sourcify. https://github.com/argotorg/sourcify/pull/2557

**Pull request merged**